app/src/test/java/com/realityexpander/tasky/domain/validation/IValidateEmailTest.kt

```diff
@@ -11,30 +12,56 @@ class IValidateEmailTest {
 
     private lateinit var validateEmail: IValidateEmail
     private val emailMatcherFakeImpl = EmailMatcherFakeImpl()
+    private val emailMatcherRegexImpl = EmailMatcherRegexImpl()
 
-    @Test
+    @Test // NOTE: we are testing the fake implementation (bc the prod impl uses android.util.Patterns.EMAIL_ADDRESS)
     fun `validateEmail() returns true for valid email`() {
 
         // ARRANGE
         validateEmail = ValidateEmailImpl(emailMatcherFakeImpl)
 
-        // ACT
-        val result = validateEmail.validate("chris@demo.com")
+        // ACT / ASSERT
+        assertTrue(validateEmail.validate("chris@demo.com")) // valid email
 
-        // ASSERT
-        assertTrue(result)
     }
```

Comment:
What's the point of adding that test case if you actually just test your fake implementation?

Reply:
I just want to be sure my fake implementation is actually working properly. Is this a bad idea?